docs/changehistory/NextVersion.md

```diff
@@ -3,10 +3,50 @@
 Table of contents:
 
 - [@itwin/appui-react](#itwinappui-react)
+  - [Deprecations](#deprecations)
   - [Changes](#changes)
 
 ## @itwin/appui-react
 
+### Deprecations
+
+- Deprecated `SignInProps` instead use `React.ComponentProps<typeof SignIn>`. [#991](https://github.com/iTwin/appui/pull/991)
```

Comment:
Let's join these under a single bullet point (per package) as it is repetitive and bloats the changelog.

Reply:
Should I still mention what props where deprecated? Or just summarize that various props were deprecated in favor of `ComponentProps`?

Reply:
Grouped them by packages.